solr/modules/ltr/src/java/org/apache/solr/ltr/model/MultipleAdditiveTreesModel.java

```diff
@@ -396,26 +505,53 @@ private static String explainNode(float[] featureVector, RegressionTreeNode regr
   // -10.0 = tree 1 | val: -10.0
   @Override
   public Explanation explain(
-      LeafReaderContext context, int doc, float finalScore, List<Explanation> featureExplanations) {
-    final float[] fv = new float[featureExplanations.size()];
-    int index = 0;
-    for (final Explanation featureExplain : featureExplanations) {
-      fv[index] = featureExplain.getValue().floatValue();
-      index++;
+          LeafReaderContext context, int doc, float finalScore, List<Explanation> featureExplanations) {
```

Comment:
could not create another explain method in LTRScoringModel otherwise, other models must override it too